### PR TITLE
Update platform_index.rst

### PR DESCRIPTION
**Pull request by jgroom33**
##### SUMMARY

Add Ciena SAOS 6 OS

##### ISSUE TYPE

- Docs Pull Request

+label: docsite_pr

##### SUMMARY

Add Ciena SAOS 6 OS to collection docs

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

Ciena SAOS 6 OS

### CODE REVIEW

**Review by samccann — APPROVED** [review]

File: docs/docsite/rst/network/user_guide/platform_index.rst [hunk 1]
```diff
@@ -64,6 +64,7 @@ Settings by Platform
     `Cisco IOS`_ `[†]`_                 ``ios``                  ✓                              ✓
     `Cisco IOS XR`_ `[†]`_              ``iosxr``                ✓                              ✓
     `Cisco NX-OS`_ `[†]`_               ``nxos``                 ✓                     ✓        ✓
+    `Ciena SAOS6`_                      ``saos6``                ✓                              ✓
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/platform_index.rst]: Hi @jgroom33 and thanks for the Ansible documentation PR. Can you please move this entry above all the Cisco entries so the table stays in alphabetical order.

Comment (jgroom33) [reply docs/docsite/rst/network/user_guide/platform_index.rst]: good catch... yes

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @jgroom33 !  Also consider whether this collection could benefit from a platform guide like some of the others in that page have.

**Comment by jgroom33** [comment]
@samccann what is the purpose/benefit of a platform guide?

**Comment by samccann** [comment]
@jgroom33 it just gives a quick summary of how to connect to the platform and an example task. See https://docs.ansible.com/ansible/devel/network/user_guide/platform_cnos.html for an idea.

**Pull request merged**